app/script/conversation/ConversationRepository.coffee

```diff
@@ -2484,32 +2455,6 @@ class z.conversation.ConversationRepository
     .then ->
       return event_json
 
-
-  ###############################################################################
-  # Helpers
-  ###############################################################################
-
-  ###
-  Archive all conversations but not the self conversation.
-  @note Archiving the self conversation will lead to problems on other clients (like Android).
-  ###
-  archive_all_conversations: =>
-    @archive_conversation conversation_et for conversation_et in @conversations() when not conversation_et.is_self()
```

Comment:
Why are we removing these helpers? I use them regularly. Can we move them to DebugUtil?